contrib/gitian-debian/postinst

```diff
@@ -42,8 +47,18 @@ db_input high bitcoinxt/select_network || true
 db_go || true
 db_get bitcoinxt/select_network
 if [ "$RET" = "Bitcoin (BTC)" ]; then
-    echo "uahftime=0" >> /etc/bitcoinxt/bitcoin.conf
-    echo "BTC Network: uahftime=0 added to /etc/bitcoinxt/bitcoin.conf"
+    if grep -q '^uahftime' $BITCOIN_CONF; then
+        sed -i 's/^uahftime.*/uahftime=0/' $BITCOIN_CONF
```

Comment:
Why not match ^uahftime=.* similar to the matching on rpcpassword?

Reply:
For consistency `grep -q '^uahftime'` could be `grep -q '^uahftime='`. I'd have to re-test all the execution paths again, so I don't think it's worth changing.